1-exercises/H-array-methods-2/exercise.js

```diff
@@ -14,9 +14,8 @@ let everyone = [
   "Austine",
   "Swathi",
 ];
-
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(0, 5); // complete this statement
+let lastFive = everyone.slice(2, 7);
```

Comment:
This would work for this case, but if you start adding new names to everyone then it would fail. You could also do a more general solution like:

`let lastFive = everyone.slice(-5);`